js/client/app/todoList/updateCountText.js

```diff
@@ -0,0 +1,16 @@
+export function updateCountText(userID) {
+  /**
+   * Call this function AFTER filter is applied.
+   */
+  const userLI = document.getElementById(userID);
+  const todos = userLI.querySelector("section.main ul.todo-list").children;
+  const countText = userLI.querySelector(
+    "div.count-container span.todo-count strong"
+  );
+
+  let count = 0;
+  for (let index = 0; index < todos.length; index++) {
+    if (todos.item(index).style.display !== "none") count++;
+  }
+  countText.textContent = count;
+}
```

Comment:
```suggestion
  const todosArray = Array.from(todos);
  countText.textContent = todosArray.filter($item => $item.style.display !== "none").length;
```

이렇게 바꾸면 더 간결하고 보기 좋지 않을까요?